ts/screens/authentication/CiePinScreen.tsx

```diff
@@ -0,0 +1,132 @@
+import { Button, H2, Text, View } from "native-base";
```

Comment:
Put a comment describing what this screen does
Something like [this](https://github.com/teamdigitale/io-app/blob/8b6fafac1e611550ce1d8eeb55f26339cdbe945d/ts/screens/onboarding/EmailInsertScreen.tsx#L1)